tests/unit/monitor/data_monitoring/test_filters_schema.py

```diff
@@ -0,0 +1,254 @@
+import pytest
+
+from elementary.monitor.data_monitoring.schema import FiltersSchema
+
+
+def test_empty_from_cli_params():
+    cli_filter = ()
+    filter_schema = FiltersSchema.from_cli_params(cli_filter)
+    assert len(filter_schema.tags) == 0
+    assert len(filter_schema.owners) == 0
+    assert len(filter_schema.models) == 0
+    assert len(filter_schema.statuses) == 1
+    assert sorted(filter_schema.statuses[0].values) == sorted(
+        ["fail", "error", "runtime error", "warn"]
+    )
+    assert len(filter_schema.resource_types) == 0
+
+
+def test_tags_key_from_cli_params():
+    cli_filter = ("tags:tag1",)
+    filter_schema = FiltersSchema.from_cli_params(cli_filter)
+    assert len(filter_schema.tags) == 1
+    assert filter_schema.tags[0].values == ["tag1"]
+    assert len(filter_schema.owners) == 0
+    assert len(filter_schema.models) == 0
+    assert len(filter_schema.statuses) == 1
+    assert sorted(filter_schema.statuses[0].values) == sorted(
+        ["fail", "error", "runtime error", "warn"]
+    )
+    assert len(filter_schema.resource_types) == 0
+
+    cli_filter = ("tags:tag1,tag2",)
+    filter_schema = FiltersSchema.from_cli_params(cli_filter)
+    assert len(filter_schema.tags) == 1
+    assert sorted(filter_schema.tags[0].values) == sorted(["tag1", "tag2"])
```

Comment:
so this means that it's multiple values (OR) and the test below is for AND?
Not sure if you did but it can be nice to add a comment explaining it in the FilterSchema class